src/core/components/user-feedback/package.json

```diff
@@ -30,20 +34,11 @@
     "@guardian/src-icons": "^3.3.0-rc.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.10",
-    "@babel/preset-env": "^7.10.0",
-    "@babel/preset-react": "^7.10.0",
-    "@babel/preset-typescript": "^7.9.0",
-    "@emotion/babel-preset-css-prop": "^11.0.0",
-    "@guardian/src-foundations": "^3.3.0-rc.0",
-    "rollup": "^1.17.0",
-    "rollup-plugin-babel": "^4.3.3",
-    "rollup-plugin-commonjs": "^10.0.2",
-    "rollup-plugin-node-resolve": "^5.2.0"
+    "npm-run-all": "^4.1.5",
+    "typescript": "^4.1.3"
   },
   "peerDependencies": {
     "@emotion/react": "^11.1.2",
-    "@guardian/src-foundations": "^3.3.0-rc.0",
-    "react": "^17.0.1"
```

Comment:
Just to clarify, are we saying that consumers no longer need to bring React in order to get Source working?